straxen/plugins/veto_events.py

```diff
@@ -427,3 +427,51 @@ def first_hitlets(hitlets_per_event: np.ndarray,
         res_hitlets_in_event.append(h)
         res_n_prompt[ind] = len(h)
     return res_hitlets_in_event, res_n_prompt
+
+
+@strax.takes_config(
+    strax.Option('event_left_extension_mv', default=0,
+                 child_option=True, parent_option_name='event_left_extension_nv',
+                 help="Extends events this many ns to the left"),
+    strax.Option('event_resolving_time_mv', default=300,
+                 child_option=True, parent_option_name='event_resolving_time_nv',
+                 help="Resolving time for fixed window coincidence [ns]."),
+    strax.Option('event_min_hits_mv', default=3,
+                 child_option=True, parent_option_name='event_min_hits_nv',
+                 help="Minimum number of fully confined hitlets to define an event."),
+    strax.Option('gain_model_mv',
+                 child_option=True, parent_option_name='gain_model_nv',
+                 help='PMT gain model. Specify as (model_type, model_config)'),
```

Comment:
Can we add a default, now I am not sure what happens?

Reply:
You will get as in the normal cases the gain settings as specified in the context: 
```
{'events_mv': ('muVETOEvents',
  '0.0.1',
  {'gain_model_mv': ('to_pe_constant', 'adc_mv'),
   'event_left_extension_mv': 0,
   'event_resolving_time_mv': 300,
   'event_min_hits_mv': 3,
   'nVETOEvents': '0.0.2'})
```
But I can also add a default if you like.

Reply:
yeah, would be nice, as far as Jelle and I agreed, this was the convention:
https://github.com/XENONnT/straxen/pull/436#issuecomment-820949904

Reply:
Okay I was not aware of this, but it is changed now. Also for the other plugins.

Reply:
ok, thanks 😄 , I guess that means we can remove it from contexts.py?